tools/pipelines/deploy-website.yml

```
@@ -1,43 +1,30 @@
 # Copyright (c) Microsoft Corporation and contributors. All rights reserved.
 # Licensed under the MIT License.
 
-# build-docs pipeline
-# This pipeline builds the main branch docs as well as previous major release branch docs each time the client packages are built.
-# Note that the method for getting the api json for main differs from previous versions. For main, the pipeline bundles
-# the published artifacts from other pipeline resources and publishes an artifact to the pipeline which can be later downloaded.
-# However, for previous version branches (in this case: 1.0) cannot run on the pipeline to generate resources. Thus, api-extractor
-# was run on 1.0 locally and manually published to the azure storage as "lts-1.0.tar.gz" which is later downloaded using
-# the AzureCLI@2 task.
-# For now, the only previous version which is manually uploaded is lts. However, for any previous versions which cannot run on the
-# pipeline, we assume that the api json content would be uploaded to the azure storage in the format: latest-${{version}}.tar.gz
+# deploy-website pipeline
+# This pipeline builds the Fluid Framework documentation website and deploys it to Azure Static Web Apps.
+# The api-model artifacts are downloaded from the azure blob storage and used to generate the markdown files.
+# The site build artifacts are published as a pipeline artifact and then deployed to Azure Static Web Apps.
 
 name: $(Build.BuildId)
 
 parameters:
-  - name: deployOverride
-    displayName: Deployment Override (default = based on branch)
-    type: string
-    default: default
-    values:
-      - default
-      - skip
-      - force
-  - name: deployEnvironment
+  - name: deployEnvironment # determines deployment token to be used for Azure Static Web Apps
```

Nit: I would recommend putting the comments on the lines above
```suggestion
  # determines deployment token to be used for Azure Static Web Apps
  - name: deployEnvironment 
```